### PR TITLE
Phalcon routes fix

### PR DESCRIPTION
**Pull request by TheGlenn88**
# FEATURE_NAME

#### DESCRIPTION
Phalcon Routes do not work by default.

### CODE REVIEW

File: docs/examples/setup-phalcon.rst [hunk 1]
```diff
@@ -160,3 +161,27 @@ host operating systems ``/etc/hosts`` file (or ``C:\Windows\System32\drivers\etc
 Open your browser at http://my-phalcon.loc or https://my-phalcon.loc
 
 .. seealso:: :ref:`setup_valid_https`
+
+
+7. Create custom vhost config file (Nginx Only)
+--------------------
```
Comment (cytopia) [inline docs/examples/setup-phalcon.rst]: This line must be as long as the headline itself

File: docs/examples/setup-phalcon.rst [hunk 2]
```diff
@@ -160,3 +161,27 @@ host operating systems ``/etc/hosts`` file (or ``C:\Windows\System32\drivers\etc
...
+
+By default routes will not work if using Nginx. To fix this, you will need to create a custom vhost configuration.
+
+In your project folder, you will need to create a folder called .devilbox unless you changed `HTTPD_TEMPLATE_DIR` in your .env
+
+Copy the default nginx config from ./cfg/vhost-gen/nginx.yml-example to ./data/www/my-project/.devilbox/nginx.yml
+
+Carefully edit the nginx.yml file and change:
+
+try_files $uri $uri/ /index.php$is_args$args;
```
Comment (cytopia) [inline docs/examples/setup-phalcon.rst]: Could you wrap this into two backticks, so its interpreted as code

```
``try_files $uri $uri/ /index.php$is_args$args;``
```

File: docs/examples/setup-phalcon.rst [hunk 3]
```diff
@@ -160,3 +161,27 @@ host operating systems ``/etc/hosts`` file (or ``C:\Windows\System32\drivers\etc
...
+to
+try_files $uri $uri/ /index.php?_url=$uri&$args;
```
Comment (cytopia) [inline docs/examples/setup-phalcon.rst]: Please see above for inline code

File: docs/examples/setup-phalcon.rst [hunk 4]
```diff
@@ -160,3 +161,27 @@ host operating systems ``/etc/hosts`` file (or ``C:\Windows\System32\drivers\etc
...
+
+and
+
+location ~ \.php?$ {
```
Comment (cytopia) [inline docs/examples/setup-phalcon.rst]: Please see above for inline code

File: docs/examples/setup-phalcon.rst [hunk 5]
```diff
@@ -160,3 +161,27 @@ host operating systems ``/etc/hosts`` file (or ``C:\Windows\System32\drivers\etc
...
+to
+location ~ [^/]\.php(/|$) {
```
Comment (cytopia) [inline docs/examples/setup-phalcon.rst]: Please see above for inline code

**Comment by cytopia** [comment]
@TheGlenn88 almost fixed. The title underline is still missing. See open comment above and the Travis check here: https://travis-ci.org/cytopia/devilbox/jobs/452282173#L701

**Comment by TheGlenn88** [comment]
I have now made the underline the same length as the title. Apologies, took me a moment to work out what was required.

**Pull request merged**